routes/error-tracker.js

```diff
@@ -20,11 +20,11 @@
 
 const winston = require('winston');
 const statusCodes = require('http-status-codes');
+const decodeURIComponent = require('safe-decode-uri-component');
```

Comment:
`safeDecodeURIComponent`? Let's not shadow the native `decodeURIComponent` for better readability.